docs/developer/components/configure.mdx

```diff
@@ -0,0 +1,161 @@
+---
+title: 'Configure'
+date: 2024-04-19T11:00:00+00:00
+sidebar_position: 3
+draft: false
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Components can be passed a list of configuration at runtime to use when executing. This configuration should first be defined using
```

Comment:
Looks like there's a missing conclusion to the second sentence on this line

Reply:
Good catch! Got all caught up in adding the following section 😄